test/e2e/apps/rc.go

```diff
@@ -558,3 +563,55 @@ func updateReplicationControllerWithRetries(c clientset.Interface, namespace, na
 	}
 	return rc, pollErr
 }
+
+type rcPatchFunc func(pdb *v1.ReplicationController) ([]byte, error)
+
+func patchRCOrDie(cs kubernetes.Interface, dc dynamic.Interface, ns string, name string, f rcPatchFunc, subresources ...string) (updated *v1.ReplicationController, err error) {
+	err = retry.RetryOnConflict(retry.DefaultRetry, func() error {
```

Comment:
just double checking if the default retry backoff is sufficient here, as it's quite short (40-50ms).
i don't know how patchRCOrDie will be used, but depending on the N conflicts, this might have to be increased.